tasks/post_copy_request_to_queue/post_copy_request_to_queue.py

```diff
@@ -239,10 +232,8 @@ def query_db(key_path: str, bucket_name: str) -> List[Dict[str, str]]:
 def get_metadata_sql(key_path: str) -> text:
     """
     Query for finding metadata based on key_path and PENDING status.
-
```

Comment:
Not sure why you are removing these empty lines. They provide whitespace making the description easier to read.

Reply:
added back